software/bsg_manycore_lib/bsg_group_strider.hpp

```diff
@@ -0,0 +1,74 @@
+// The intent of this file is to provide a simple, templated method
```

Comment:
Is this class an interface for a tile accessing memory that is S_X to the west and and S_Y to the south, where S_X and S_Y are static?

Reply:
East, but yes. 

You can make them negative to move in the other direction.

Reply:
It ends up turning into ~2-3 instructions to move between tile groups.

Reply:
what happens if you're at the eastern or southern edge of a tile-group?

Reply:
well... could if you use -1 for S_X, can't you access east not west?

Reply:
Positive moves in the South (vertical) and East (Horizontal) directions. They wrap around when the edge of the tile group is reached.

Reply:
Neat!